src/test/kotlin/org/jetbrains/kotlinx/jupyter/test/repl/AbstractReplTest.kt

```diff
@@ -58,6 +59,14 @@ abstract class AbstractReplTest {
         return ReplForJupyterImpl(resolutionInfoProvider, embeddedClasspath, isEmbedded = true)
     }
 
+    protected fun Map<String, VariableState>.getStringValue(variableName: String): String? {
+        return get(variableName)?.stringValue
+    }
+
+    protected fun Map<String, VariableState>.getValue(variableName: String): Any? {
+        return get(variableName)?.value
+    }
+
```

Comment:
Let's replace these declarations to `testUtil.kt`, they may be useful in other tests as well